tests/assistant_spec.js

```diff
@@ -62,7 +62,7 @@ const graphieLink6 = makeGraphieLink();
 
 const getEnglishStr = (item) => item.englishStr;
 const getTranslation = (item) => item.translatedStr;
-const lang = 'fr';
+const lang = 'cs';
```

Comment:
Since this is the default, how about we handle it within `assertSuggestions` itself, and remove this file level const?

If we change `assertSuggestions` signature to:

```
function assertSuggestions(allItems, itemsToTranslate, translatedStrs,
        lang = 'cs') {
```

Then in any test case that's currently using this default language we can simply omit passing in `lang` into `assertSuggestions` so it defaults to `cs`?